public/electron/main.js

```diff
@@ -416,7 +415,21 @@ function handleEventSend(code) {
         break;
       case 2:
         // User selects "Continue Anyway", we must be in dev mode
-        CONTINUE_ANYWAY = true;
+        // resolve variable unused warning
+        if (CONTINUE_ANYWAY == false) {
+          CONTINUE_ANYWAY = true;
+        }
```

Comment:
I believe this can be simplified without the if statement

```suggestion
        CONTINUE_ANYWAY = true;
```

Reply:
Is this mainly a statement to use the `CONTINUE_ANYWAY` variable?

Reply:
I agree this is very redundant -- yes, it is primarily to use this variable to pass the eslint tests. I was wondering since it gives this unused variable warning, would it be okay to just delete this variable overall?

Reply:
We can safely remove the variable! That variable was only there to skip sending data to a port that didn't exist, but now we have a mock port to send it to